spec/spec_helper.rb

```diff
@@ -1,5 +1,8 @@
 require 'simplecov'
 require 'simplecov-console'
+require "database_connection"
+
+DatabaseConnection.connect("shop_database")
```

Comment:
Ideally the tests should use a different database (the "test" database) than the application when running in the terminal — this is so we don't have tests writing / updating data that is production data used by "real" users